opwen_email_server/integration/webapp.py

```diff
@@ -226,7 +227,9 @@ def _delete(self, email_address: str, uids: Iterable[str]):
             else:
                 continue
 
-            self._mailbox_storage.delete(f"{domain}/{email_address}/{folder}/{email['sent_at']}/{uid}")
+            desc_prefix = descending_timestamp(email['sent_at'])
+
+            self._mailbox_storage.delete(f"{domain}/{email_address}/{folder}/{desc_prefix} {email['sent_at']}/{uid}")
```

Comment:
Given that we no longer need the `sent_at` path piece, I'd recommend to just remove it to simplify the folder structure.

Reply:
Sounds good, I was on the fence as I was unsure if the `sent_at` portion would have been helpful when navigating the Azure folders in the event we needed to manually find something.